toolchain/llvm/llvm-compiler.in

```diff
@@ -22,6 +22,9 @@ else
             LTO_FLAGS="$LTO_FLAGS -flto-jobs=@CPU_COUNT@"
         fi
     fi
+    if [ "@LLVM_ENABLE_PGO@" == "GEN" ] || [ "@LLVM_ENABLE_PGO@" == "CSGEN" ]; then
```

Comment:
`if [[ -n "@LLVM_PROFILE_DATA_DIR@" ]]; then`


When training llvm, `LLVM_ENABLE_PGO` is meaningless, no need to set it to `GEN` or `CSGEN`.
 After this modification, training LLVM when building toolchain or packages does not require additional setting of cmake variables. And just set `LLVM_PROFILE_DATA_DIR` when you need to override the default profile data location.

Reply:
No conflict with `LLVM_PROFDATA_FILE` used with `LLVM_ENABLE_PGO=USE`.